### DISCUSSION TITLE
Build instructions, anyone?

### DISCUSSION THREAD

**Discussion by macruspareto** [sandboxie-plus/Sandboxie#1728] [Q&A Feedback]
.

**Comment by isaak654** [comment]
Unfortunately there wasn't a lot of pull request activity about that, we would need more volunteers to cover this aspect.

For this reason, I made a specific "build issue" label in order to collect all build requests and replies:
https://github.com/sandboxie-plus/Sandboxie/issues?q=label%3A%22build+issue%22

It is clearly mentioned in the README.md file located in the root of the repository, under the entry
`We recommend to check also the following labels in this repository`

I understand it's not the most complete solution, but it's obviously better than nowhere.

EDIT: There is also https://github.com/sandboxie-plus/Sandboxie/discussions/1089
